velox/functions/prestosql/tests/ArrayContainsTest.cpp

```diff
@@ -26,14 +27,24 @@ using namespace facebook::velox::functions::test;
 
 namespace {
 
-class ArrayContainsTest : public FunctionBaseTest {};
-
-TEST_F(ArrayContainsTest, integerNoNulls) {
-  auto arrayVector = makeArrayVector<int64_t>(
-      {{1, 2, 3, 4}, {3, 4, 5}, {}, {5, 6, 7, 8, 9}, {7}, {10, 9, 8, 7}});
+class ArrayContainsTest : public FunctionBaseTest {
+ public:
+  void testContains(
+      ArrayVectorPtr arrayVector,
+      std::vector<int64_t> search,
+      const std::vector<std::optional<bool>>& expected) {
+    auto constSearch = BaseVector::wrapInConstant(
+        1, 0, makeArrayVector<int64_t>({std::move(search)}));
+    auto result = evaluate<SimpleVector<bool>>(
```

Comment:
nit: I have seen this form a lot, whats the purpose of it evaluate<SimpleVector<bool>> -> evaluate(..)?
anyway we assert vector equivalence after that.

Reply:
We do not need it. Fixed it by using evaluate directly.